src/frankenphp-symfony/src/Runner.php

```diff
@@ -22,22 +22,25 @@ public function __construct(private HttpKernelInterface $kernel)
 
     public function run(): int
     {
+        $kernel = $this->kernel;
         $server = array_filter($_SERVER, static fn (string $key) => !str_starts_with($key, 'HTTP_'), ARRAY_FILTER_USE_KEY);
         $server['APP_RUNTIME_MODE'] = 'web=1&worker=1';
 
-        do {
-            $ret = \frankenphp_handle_request(function () use ($server, &$sfRequest, &$sfResponse): void {
-                // Merge the environment variables coming from DotEnv with the ones tight to the current request
-                $_SERVER += $server;
+        $handler = static function () use ($kernel, $server, &$sfRequest, &$sfResponse): void {
+            // Merge the environment variables coming from DotEnv with the ones tight to the current request
```

Comment:
Typo? Not sure "tight" made sense here :thinking: 

```suggestion
            // Merge the environment variables coming from DotEnv with the ones tied to the current request
```